eisbuk-admin/src/components/atoms/AttendanceCard/AttendanceCard.tsx

```diff
@@ -37,12 +41,19 @@ const AttendanceCard: React.FC<Props> = ({
       <div data-testid="time-string">{timeString}</div>
       <div>{categories}</div>
       {userBookings.map((user) => {
+        const isAbsent = absentees?.includes(user.customer_id) || false;
         return (
           <div key={user.customer_id}>
             <div>{user.name}</div>
-            <div>
-              {absentees?.includes(user.customer_id) ? "absent" : "present"}
-            </div>
+            <div>{isAbsent ? "👎" : "👍"}</div>
+            <button
+              type="button"
+              onClick={() =>
+                dispatch(markAttendance(user.customer_id, isAbsent))
+              }
+            >
+              {`Mark ${user.name} as ${!isAbsent ? "absent" : "present"}`}
+            </button>
```

Comment:
Ah, so...we're clicking on emojis, right. Not the button with the text. 
Should be just like the original.